experiments/ClimaCore/sea_breeze/run.jl

```diff
@@ -61,115 +83,96 @@ ocn_Y_default, ocn_domain = ocn_init(xmin = -500, xmax = 0, helem = 10, npoly =
 
 lnd_Y_default, lnd_domain = lnd_init(xmin = 0, xmax = 500, helem = 10, npoly = 0)
 
+# Build remapping operators
 atm_boundary = Spaces.level(atm_domain.hv_face_space, PlusHalf(0))
 
-# Build remapping operators
 atm_to_ocn = Operators.LinearRemap(ocn_domain, atm_boundary)
 atm_to_lnd = Operators.LinearRemap(lnd_domain, atm_boundary)
 ocn_to_atm = Operators.LinearRemap(atm_boundary, ocn_domain)
 lnd_to_atm = Operators.LinearRemap(atm_boundary, lnd_domain)
 
-function make_dss_func()
-    function _dss!(x::Fields.Field)
-        Spaces.weighted_dss!(x)
-    end
-    function _dss!(::Any)
-        nothing
-    end
-    dss_func(Y, t, integrator) = foreach(_dss!, Fields._values(Y))
-    return dss_func
+# initialize coupling fields
+atm_T_sfc = Operators.remap(ocn_to_atm, ocn_Y_default.T_sfc) .+ Operators.remap(lnd_to_atm, lnd_Y_default.T_sfc) # masked arrays; regrid to atm grid
+atm_F_sfc = Fields.zeros(atm_boundary)
+ocn_F_sfc = Fields.zeros(ocn_domain)
+lnd_F_sfc = Fields.zeros(lnd_domain)
+
+# init models
+atm_Y = Fields.FieldVector(Yc = atm_Y_default.Yc, ρw = atm_Y_default.ρw, F_sfc = atm_F_sfc)
+atm_p = (cpl_p = cpl_parameters, T_sfc = atm_T_sfc, bc = atm_bc)
+atmos = AtmosSimulation(atm_Y, t_start, cpl_Δt / atm_nsteps, t_end, SSPRK33(), atm_p, saveat, dss_callback)
+
+ocn_Y = Fields.FieldVector(T_sfc = ocn_Y_default.T_sfc)
+ocn_p = (cpl_parameters, F_sfc = ocn_F_sfc)
+ocean = OceanSimulation(ocn_Y, t_start, cpl_Δt / ocn_nsteps, t_end, SSPRK33(), ocn_p, saveat)
+
+lnd_Y = Fields.FieldVector(T_sfc = lnd_Y_default.T_sfc)
+lnd_p = (cpl_parameters, F_sfc = lnd_F_sfc)
+land = LandSimulation(lnd_Y, t_start, cpl_Δt / lnd_nsteps, t_end, SSPRK33(), lnd_p, saveat)
+
+# coupled simulation
+struct AOLCoupledSimulation{FT, A, O, L} <: ClimaCoupler.AbstractCoupledSimulation
+    atmos::A
+    ocean::O
+    land::L
+    Δt::FT
```

Comment:
I wonder if `Δt` should live in the `CoupledSimulation`. Once we start running the AOL models at different time steps, their respective `Δt` will live in their integrator.

Reply:
This is the coupled time step; as you note, each individual simulation has its own time step that will live in the integrators. I can add a in-code comment to clarify this.